src/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+</head>
+<body>
+    <header>
+        <h1>Leaderboard</h1>
+    </header>
+    
+    <section class="main">
+        <section class="scores">
+            <div class="scores-head">
+                <h2>Recent scores</h2>
+                <button type="button">Refresh</button>
+            </div>
+            <ul class="scores-list">
+                <li>name: 100</li>
+                <li>name: 100</li>
+                <li>name: 100</li>
+                <li>name: 100</li>
+            </ul>
+
+        </section>
+        <section class="add-form">
+            <h2 class="form-title">Add your score</h2>
+            <form action="" class="form">
+                <input type="text" class="input-field" placeholder="Your name">
+                <input type="text" class="input-field" placeholder="Your score">
+                <button type="submit" class="submit-button">Submit</button>
+            </form>
+        </section>
+        
+    </section>
```

Comment:
- [ ] @sanieni6 great work on your form, however, its design doesn't match the wireframe given, the title width should be equal to the form width, and the button should be shorter and aligned to the right, also the refresh button should be aligned to the right check the comparison below


Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/223538738-6c2fd899-d18e-4c92-8193-0ca2adfa0dcb.png) | ![image](https://user-images.githubusercontent.com/61408860/223538809-ca0685d7-f9e5-4b52-82ea-052a8ed69bd8.png)